notebooks/WatsonNLP_Demo.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>Is this string an authentication token?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>